### PR TITLE
Created subject level component

### PR DESCRIPTION
**Pull request by MickhaelSh**
Created subject level component, fixed adaptivity in coop page, minor fixes
image
image

### CODE REVIEW

File: tests/unit/components/offer-banner/OfferBanner.spec.jsx [hunk 1]
```diff
@@ -13,12 +13,12 @@ const buttonActions = [
 const mockOffer = {
   _id: 'id',
   authorAvgRating: 4.3,
```
Comment (tsivkadi) [inline tests/unit/components/offer-banner/OfferBanner.spec.jsx:15]: we don`t have this field anymore either, so you can remove it

File: tests/unit/components/offer-banner/OfferBanner.spec.jsx [hunk 2]
```diff
@@ -13,12 +13,12 @@ const buttonActions = [
...
-  authorLastName: 'Wilson',
   description:
     'Hello. There are many variations of passages of Lorem Ipsum available, but the majority have suffered alteration in some form, by injected humour, or randomised words which.',
   languages: ['Ukrainian', 'English'],
   author: {
+    firstName: 'James',
+    lastName: 'Wilson',
     totalReviews: {
       student: 0,
       tutor: 0
```
Comment (tsivkadi) [inline tests/unit/components/offer-banner/OfferBanner.spec.jsx:24]: instead, add averageRating field here

File: src/components/offer-banner/OfferBanner.tsx [hunk 3]
```diff
@@ -1,18 +1,18 @@
...
 import IconButton from '@mui/material/IconButton'
 import TurnedInNot from '@mui/icons-material/TurnedInNot'
-import { useTranslation } from 'react-i18next'
 
 import useBreakpoints from '~/hooks/use-breakpoints'
-import AppChip from '~/components/app-chip/AppChip'
 import AppCard from '~/components/app-card/AppCard'
 import AppButton from '~/components/app-button/AppButton'
 import UserProfileInfo from '~/components/user-profile-info/UserProfileInfo'
+import SubjectLevelChips from '../subject-level-chips/SubjectLevelChips'
```
Comment (Yurenko) [inline src/components/offer-banner/OfferBanner.tsx:13]: Use alias

File: src/components/subject-level-chips/SubjectLevelChips.tsx [hunk 4]
```diff
@@ -0,0 +1,44 @@
+import { FC, useMemo } from 'react'
+import Box from '@mui/material/Box'
+import { SxProps } from '@mui/material'
+import AppChip from '~/components/app-chip/AppChip'
```
Comment (Yurenko) [inline src/components/subject-level-chips/SubjectLevelChips.tsx:4]: add space between imports

File: src/containers/my-cooperations/cooperations-container/CooperationContainer.constants.tsx [hunk 5]
```diff
@@ -1,28 +1,38 @@
-import Box from '@mui/material/Box'
-import StatusChip from '~/components/status-chip/StatusChip'
-import AppChip from '~/components/app-chip/AppChip'
 import { getFormatedDate } from '~/utils/helper-functions'
+import StatusChip from '~/components/status-chip/StatusChip'
+import UserProfileInfo from '~/components/user-profile-info/UserProfileInfo'
+import SubjectLevelChips from '~/components/subject-level-chips/SubjectLevelChips'
 import { Cooperation } from '~/types'
 import { styles } from '~/containers/my-cooperations/cooperations-container/CooperationContainer.styles'
+import { Typography } from '@mui/material'
```
Comment (Yurenko) [inline src/containers/my-cooperations/cooperations-container/CooperationContainer.constants.tsx:7]: Move above and do it like this `import Typography from '@mui/material'/Typography `

File: src/constants/translations/ua/cooperations-page.json [hunk 6]
```diff
@@ -13,5 +13,6 @@
     "price": "Ціна",
     "updated": "Останнє оновлення",
     "status": "Статус"
-  }
+  },
+  "search": "Пошук за ім'ям, або заголовком"
```
[the remark this answers is not in the document]
Comment (MickhaelSh) [reply src/constants/translations/ua/cooperations-page.json]: fixed

File: tests/unit/components/offer-banner/OfferBanner.spec.jsx (hunk 1 above)
Comment (MickhaelSh) [reply tests/unit/components/offer-banner/OfferBanner.spec.jsx]: fixed

File: tests/unit/components/offer-banner/OfferBanner.spec.jsx (hunk 2 above)
Comment (MickhaelSh) [reply tests/unit/components/offer-banner/OfferBanner.spec.jsx]: fixed

File: src/containers/my-cooperations/cooperations-container/CooperationContainer.constants.tsx (hunk 5 above)
Comment (MickhaelSh) [reply src/containers/my-cooperations/cooperations-container/CooperationContainer.constants.tsx]: fixed

File: src/components/subject-level-chips/SubjectLevelChips.tsx (hunk 4 above)
Comment (MickhaelSh) [reply src/components/subject-level-chips/SubjectLevelChips.tsx]: fixed

File: src/components/offer-banner/OfferBanner.tsx (hunk 3 above)
Comment (MickhaelSh) [reply src/components/offer-banner/OfferBanner.tsx]: fixed

**Review by Yurenko — APPROVED** [review]

**Pull request merged**